redisinsight/ui/src/components/side-panels/panels/enablement-area/EnablementArea/components/Group/Group.styles.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Theme } from 'uiSrc/components/base/theme/types'
+
+export const GroupHeaderButton = styled.div<
+  React.HTMLAttributes<HTMLDivElement>
+>`
+  display: flex;
```

Comment:
nit: We can use our layout components `Row`/`Col`/`FlexGroup` instead of a regular `div` and pass `align`/`justify` instead of declaring them here.